src/clientsettingsdlg.cpp

```diff
@@ -198,7 +201,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                      "In this case the buffer delay setting is disabled and has to be "
                                      "changed using the sound card driver. On Windows, press the "
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
-                                     "use the Jack configuration tool to change the buffer size." ) +
+                                     "use the JACK configuration tool to change the buffer size." ) +
                                 "<br>" +
                                 tr ( "If no buffer size is selected and all settings are disabled, an "
                                      "unsupported buffer size is used by the driver. The application "
```

Comment:
When it says "the application" does that mean the driver?

Reply:
Probably JACK/ASIO-driver?

Reply:
Hmm. No. Probably Jamulus. Changing that...